<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daniel Cantar√≠n</author>
 <class>Bandeja</class>
 <widget class="QGroupBox" name="Bandeja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>378</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>378</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>292</width>
    <height>378</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Jura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bandeja</string>
  </property>
  <property name="title">
   <string>Bandeja</string>
  </property>
  <property name="flat">
   <bool>false</bool>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="lista">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCantidadArchivos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0 archivos</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAgregarArchivos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Agregar archivos . . . </string>
   </property>
   <property name="text">
    <string> + </string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuitarArchivos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string> - </string>
   </property>
  </widget>
  <widget class="QSlider" name="volumen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>31</width>
     <height>111</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="vumetter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>70</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdRemaining">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonPlay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStop">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInterprete">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTema">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tema:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
